src/components/LoginPanel.tsx

```diff
@@ -29,11 +29,11 @@ const LoginPanel: React.FC<{}> = () => {
                     variables: {
                         default: {
                             colors: {
-                                inputLabelText: currentTheme.foregroundColor,
-                                anchorTextColor: currentTheme.foregroundColor,
-                                brand: currentTheme.fillColor,
-                                brandAccent: currentTheme.fillColor,
-                                brandButtonText: currentTheme.foregroundColor,
+                                inputLabelText: lightTheme.foregroundColor,
+                                anchorTextColor: lightTheme.foregroundColor,
+                                brand: lightTheme.surfaceBackgroundColor,
+                                brandAccent: lightTheme.surfaceBackgroundColor,
+                                brandButtonText: lightTheme.foregroundColor,
```

Comment:
This comment applies to everywhere that explicitly uses either `darkTheme` or `lightTheme`, but would it be best to have a global `currentTheme` variable that can then be toggled once instead of having to change the theme everywhere manually?

Reply:
Actually...nevermind.  Just seen where you change the theme in `ThemeProvider`. But still, is there a reason why the `LoginPanel` doesn't pull from the `ThemeProvider`?